_data/snippets.yml

```diff
@@ -34,3 +34,30 @@ trusty_note_no_osx: |
   > information are in our reference pages:
   >  * [Precise](/user/reference/precise/)
   >  * [Trusty](/user/reference/trusty/)
+
+contact_enterprise_support: |
```

Comment:
When you put the `##` heading *inside* the snippet, you need to be careful at what level you level you include it. Worth thinking about just including the text here instead?

Reply:
works for me! 👍